src/warden/src/raw.rs

```diff
@@ -192,6 +202,7 @@ pub struct DrawPass {
 #[derive(Debug, Deserialize)]
 pub enum Job {
     Transfer(TransferCommand),
+    Clear(ClearCommand),
```

Comment:
Let's move this into `TransferCommand` instead? Technically, it's meant to be executed on a transfer queue (or pipeline?) in Vulkan, so that where the distinction comes from